test/lib/search.bats

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bats
+
+load ../../lib/composure
+load ../../plugins/available/base.plugin
+
+cite _about _param _example _group _author _version
+
+load ../../lib/helpers
+load ../../lib/search
+
+NO_COLOR=true
+
+@test "helpers search aliases" {
+  run _bash-it-search-component 'plugins' 'base'
+  [[ "${lines[0]}" =~ 'plugins' && "${lines[0]}" =~ 'base' ]]
+}
+
+@test "helpers search all ruby et al" {
+  run _bash-it-search 'ruby' 'gem' 'bundle' 'rake' 'rails' '--disable'
+  run _bash-it-search 'ruby' 'gem' 'bundle' 'rake' 'rails'
+  [[ "${lines[0]/✓/}" == '      aliases  =>   bundler rails' ]]
+  [[ "${lines[1]/✓/}" == '      plugins  =>   chruby chruby-auto ruby' ]]
```

Comment:
These tests don't seem to work for me. When I change this line to include `ruby2` instead of `ruby`, the test still works. I expected it to fail in this case. It looks like the expression is not evaluated...